src/OpenTelemetry.Shared/ExceptionExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// <copyright file="ExceptionExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Globalization;
+using System.Threading;
+
+namespace OpenTelemetry.Shared
+{
+    internal static class ExceptionExtensions
+    {
+        /// <summary>
+        /// Returns a culture-independent string representation of the given <paramref name="exception"/> object,
+        /// appropriate for diagnostics tracing.
+        /// </summary>
+        /// <param name="exception">Exception to convert to string.</param>
+        /// <returns>Exception as string with no culture.</returns>
+        public static string ToInvariantString(this Exception exception)
```

Comment:
Are you going to move the file from https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Api/Internal/ExceptionExtensions.cs to this?

Reply:
`ExceptionExtensions` is the copy from API. We should have this in any one of the project. I don't think `OpenTelemetry.Api` will take reference of this shared project. Better solution is to take a dependency on `OpenTelemetry` or `OpenTelemetry.Api` based on other common files we get to this project. Could you please share your thoughts?

Reply:
what we could do is: instead of adding it as a direct reference, use the compile way. So, we won't have duplicated files.